cli/config/configStruct.go

```diff
@@ -33,10 +34,11 @@ type ConfigStruct struct {
 	KubeConfigPathStr      string                      `yaml:"kube-config-path"`
 	ConfigFilePath         string                      `yaml:"config-path,omitempty" readonly:""`
 	HeadlessMode           bool                        `yaml:"headless" default:"false"`
+	ServiceMap             bool                        `yaml:"service-map" default:"true"`
```

Comment:
```suggestion
	ServiceMap             bool                        `yaml:"service-map" default:"false" readonly:""`
```

Reply:
this experimental feature so the default should be false and it should not be generated to user..

Reply:
@haiut defined the flag to be true by default, can you confirm it?